wildfly-standalone-centos/README.md

```diff
@@ -22,72 +22,77 @@
+
+This Azure quickstart template deploys a web applicaton on WildFly 18.0.1.Final running on CentOS 8.
+
+The technology behind WildFly is also available in Red Hat JBoss Enterprise Application Platform (EAP), a hardended enterprise subscription with Red Hat's world-class support, long multi-year maintenance cyccles and exclusive content. JBoss EAP is an open-soure application server that you can download for free, for developement use.  To obtain a Red Hat Subscription Management (RHSM) account for JBoss EAP, go to: www.redhat.com 
 
 ## Template Solution Architecture 
 
-This template creates all of the compute resources to run WILDFLY 18.0.1 on top of CentOS 8.0, deploying the following components:
+This template creates all of the Azure compute resources to run WildFly 18.0.1.Final on top of CentOS 8.0. The following resources are created by this template:
 
 - CentOS 8 VM 
 - Public DNS 
 - Private Virtual Network 
 - Security Configuration 
 - WildFly 18.0.1.Final
-- Sample application deployed to WildFly 18.0.1.Final
+- Sample Java application deployed to WildFly
 
-Following is the Architecture :
+Following is the Architecture:
 <img src="images/wildfly.arch.PNG" width="800">
 
 To learn more about WildFly 18.0.0.Final, check out:
 https://docs.wildfly.org/18/
 
-## Licenses and Costs 
+## License, Subscription and Costs 
 
-This uses CentOS 8 image which is a PAY AS YOU GO image and doesn't require the user to license it, it will be licensed automatically after the instance is launched first time and user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details.
+This template uses an Azure CentOS 8 image which is a Pay-as-you-go (PAYG) VM image and does not require the user to license. The VM will be licensed automatically after the instance is launched for the first time and user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details. WildFly is free to download and use and does not require a Red Hat Subscription.
 
 ## Prerequisites 
 
-1. Azure Subscription with specified payment method (CentOS-Based 8.0 is a market place product and requires payment method to be specified in Azure Subscription)
+1. Azure Subscription with specified payment method (CentOS-Based 8.0 is a marketplace product and requires payment method to be specified in the user Azure Subscription).
 
-2. To create the VM, you will need to:
+2. To create the VM, you will need:
 
-    - Choose an admin user name and password for your VM.  
+    - **Admin Username** and password
 
-    - Choose a name for your VM. 
+    - **DNS Label Prefix** for the public IP
```

Comment:
Modified this line as follows. Please suggest

- DNS Label Prefix for the public IP which should be unique. Note that this will also be the name of your VM.